{
  "number": 6739,
  "title": "dotnet run on exec task exited with code 134",
  "body": "I have dotnet5 projA project which has [Exec](https://docs.microsoft.com/en-us/visualstudio/msbuild/exec-task?view=vs-2019) task which does \"dotnet run ProjectB(dotnet5 project) /some arguments\". But when I do dotnet build on ProjectA on WSL2 I'm getting \"The command \"dotnet run --project ProjectB\" exited with code 134\". However, it builds fine on Windows. I  only get this error when I do dotnet build ProjectA on WSL2. \r\n\r\nThis is my exec command: \r\n\r\n<Exec Command=\"dotnet run --project ProjectB.csproj -c Debug /treatwarningaserrors /OP=$(TargetDir) /argument1=$(ProjectDir) /argument2=test />\r\n",
  "state": "CLOSED",
  "createdAt": "2021-08-06T07:04:41Z",
  "updatedAt": "2024-02-21T14:12:12Z",
  "closedAt": "2021-08-13T11:18:05Z",
  "author": {
    "login": "doniyorniazov"
  },
  "labels": [
    "needs-more-info",
    "author-responded",
    "triaged"
  ],
  "assignees": {
    "nodes": []
  },
  "milestone": null,
  "comments": {
    "nodes": [
      {
        "body": "Team Triage: We can't reproduce this issue, could you provide a minimal project that reproduces this behavior?",
        "createdAt": "2021-08-12T17:06:16Z",
        "updatedAt": "2021-08-12T17:06:16Z",
        "author": {
          "login": "benvillalobos"
        }
      },
      {
        "body": "@BenVillalobos Thanks for your time. I couldn't reproduce this with the test project either, I think this might be specific to my project. ",
        "createdAt": "2021-08-13T11:18:05Z",
        "updatedAt": "2021-08-13T11:18:29Z",
        "author": {
          "login": "doniyorniazov"
        }
      }
    ]
  }
}